staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -146,7 +146,7 @@ func (rl *respLogger) Addf(format string, data ...interface{}) {
 // Log is intended to be called once at the end of your request handler, via defer
 func (rl *respLogger) Log() {
 	latency := time.Since(rl.startTime)
-	if glog.V(2) {
```

Comment:
drop both of these, they cancel:
```
UPSTREAM: revert: ffead55: <carry>: Fix to avoid REST API calls at log level 2.
UPSTREAM: <carry>: Fix to avoid REST API calls at log level 2.
```

Reply:
heh, we can't make up our mind... we did it, reverted it, then redid it... looks like we still want it, but should only have a single commit for it

Reply:
Dropped all but the initial carry